CHANGELOG.md

```diff
@@ -37,7 +68,7 @@
 - Fix bug where whitespace of user code in attempt wasn't shown in admin interface.
 - Fix bug where sender's email address is not listed on contact us forms sent to admin.
```

Comment:
Unrelated but I noticed this is #53 (open issue), so should the issue be resolved?